### PR TITLE
Explorer xNFT v1

### PR DESCRIPTION
**Pull request by greenish**
- adds new Explorer xNFT code (published here: https://www.xnft.gg/app/3Db1fHHc2TGrCpBWnu6ZzdQd5pSoyGCmnh6nopNjv4P2)
- gets auto-installed for all users 
- Only visible in dev-mode (same as simulator)

https://user-images.githubusercontent.com/580992/202311346-4cffb434-39d7-42c8-a3f4-c2915f50d26a.mov

### CODE REVIEW

File: packages/recoil/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "@solana/web3.js": "^1.63.1"
   },
   "devDependencies": {
+    "@coral-xyz/xnft-cli": "latest",
```
[the remark this answers is not in the document]
Comment (greenish) [reply packages/recoil/package.json:31]: oh not even sure why this is here.. will remove

**Comment by greenish** [comment]
@SocketSecurity ignore node-stdlib-browser@1.2.0

**Comment by callensm** [comment]
should this go in the `xnft-program-library` repository instead?

**Comment by greenish** [comment]
> should this go in the `xnft-program-library` repository instead?

I'm not sure exactly what `xnft-program-library` was for, but we moved the Prices xNFT from `xnft-program-library` into `backpack`

**Pull request merged**